src/Product.php

```diff
@@ -527,6 +534,14 @@ protected function getPropertyValue(array $property)
         switch ($propertyType) {
             case 'empty':
                 $value = $property['property']['backendName'];
+                if (isset($property['names'])) {
```

Comment:
You won't find the external names here. You will have to take them from the result set returned by `/items/properties`.